release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,129 @@
 
 This section highlights new features in {product} {product-version}.
 
-== Support for Role-Based Access Control 
 
-The Role-Based Access Control (RBAC) feature in {product} introduces an administrator role and optimizes access control through organizational structures such as teams, groups, and users.
+== Support for upgrading the {product} Helm Chart 
```

Comment:
This is not a notable new feature. The upgrade was supported from the beginning; there was just nothing to upgrade from, as we had only one release.

Reply:
@kadel, I understand your point. Since we did not have the upgrade procedure mentioned in the admin guide, we were planning to cover it in the Release Notes. Would it be okay if i move this piece of content in the previous chapter [about this release](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.0/html-single/release_notes_for_red_hat_developer_hub_1.0/index#con-release-notes-overview_release-notes-rhdh).